cfme/tests/openstack/cloud/test_flavors.py

```diff
@@ -125,3 +151,32 @@ def cleanup():
 
     wait_for(lambda: not flavor.exists, delay=5, timeout=600, fail_func=flavor.refresh,
              message='Wait for flavor to appear')
+
+
+@pytest.mark.rfe
+@pytest.mark.ignore_stream('5.9')
+def test_flavors_list(appliance, soft_assert, private_flavor):
+    collection = appliance.collections.cloud_flavors
+    view = navigate_to(collection, 'All')
+    item = view.entities.get_entity(name=private_flavor.name, surf_pages=True)
+    soft_assert(item.data['name'] == private_flavor.name)
+    soft_assert(item.data['cpus'] == str(private_flavor.vcpus))
+    soft_assert(item.data['publicly_available'] == str(private_flavor.is_public))
+    soft_assert(item.data['memory'].split()[0] == str(private_flavor.ram))
+    soft_assert(item.data['cloud_provider'] == private_flavor.provider.name)
+
+
+@pytest.mark.rfe
+@pytest.mark.ignore_stream('5.9')
+def test_flavor_details(appliance, soft_assert, private_flavor):
```

Comment:
this test looks very similar to above one.  maybe it makes sense to combine them. What do you think ?